### ISSUE TITLE
discordo not saving token

### ISSUE THREAD

**Issue by boatinalamp** [ayntgl/discordo#214]
^^ discordo is not saving my token, but is asking me for it every time i launch it. 
![image](https://user-images.githubusercontent.com/93244703/189281350-bc947427-507b-447a-9f3f-06b4d6462ccb.png)

**Comment by boatinalamp** [comment]
im using archcraft (arch)
******@******-inspiron3585

  · Archcraft x86_64
  · Inspiron 3585 1.4.0
  · 5.19.6-arch1-1
  · 1 hour, 54 mins
  · 1082 (pacman), 4 (flatpak)
  · zsh 5.9
  · 1920x1080
  · Gruvbox [GTK2/3]
  · Win11-Dark [GTK2/3]
  · alacritty
  · AMD Ryzen 5 2500U with Radeon Vega Mobile Gfx (8) @ 2.000GHz
  · AMD ATI Radeon Vega Series / Radeon Vega Mobile Series
  · 3959MiB / 13876MiB

**Comment by ayntgl** [comment]
Thanks for the detailed report! Could you tell me which desktop environment you are using?

**Comment by ayntgl** [comment]
> i am currently facing this issue on latest git. `-token` parameter is completely ignored and there is no help page for directions.
> 
> how will the desktop environment conflict with this? does the keyring discordo uses depend on the desktop environment?

Hello there, the `token` parameter has been removed and is no longer used by Discordo. To login using the token and save it to the default keyring, run the executable without any arguments and press `Ctrl+Space` to switch to the token login view, input your token in the input field displayed, and click on the "Login" button to continue. If the issue persists, please join our Discord server to I can troubleshoot this issue further.

**Comment by wael444** [comment]
> input your token in the input field displayed, and click on the "Login" button to continue. If the issue persists, please join our Discord server to I can troubleshoot this issue further.

is there NO possible way to automate this? atleast via an option?

**Comment by ayntgl** [comment]
You can manually create a keyring entry using a secret service frontend like Seahorse by GNOME. Create a new password keyring named "login" if it does not exist already, in the dedaupt keyring (login), create a new user named "token" and the password as the token in the next field, save it and run the executable again without any arguments, it should use the token from the keyring. Let me know if the issue still persists; join our Discord server so I can troubleshoot the issue further.

**Comment by wael444** [comment]
> You can manually create a keyring entry using a secret service frontend like Seahorse by GNOME. 

i am using a window manager so using something like this is unlikely, neither does my distro support such huge desktop environments, i will just attempt to automate this with X. would have been cool to atleast have a `token` argument..

**Comment by ayntgl** [comment]
May I know which distro you are using to run Discordo?

**Comment by wael444** [comment]
KISS

**Comment by wael444** [comment]
just allow an token option, I can make it so the token is gpg encrypted and have to unlock it and pass it to discordo.

**Comment by ayntgl** [comment]
@wael444, #245 adds flags for the authentication token, the configuration file path, and the log file path.

**Comment by wael444** [comment]
yep thankskxkdjugiy0rw7e8gqy6awhgbrje7234qweyg79t034qwy7g8t342ytg
